instrumentation/opentelemetry-instrumentation-asyncpg/src/opentelemetry/instrumentation/asyncpg/__init__.py

```diff
@@ -60,49 +60,50 @@ async def main():
 from opentelemetry.instrumentation.asyncpg.version import __version__
 from opentelemetry.instrumentation.instrumentor import BaseInstrumentor
 from opentelemetry.instrumentation.utils import unwrap
-from opentelemetry.semconv.trace import (
-    DbSystemValues,
-    NetTransportValues,
-    SpanAttributes,
+from opentelemetry.semconv._incubating.attributes.db_attributes import (
+    DB_NAME,
+    DB_STATEMENT,
+    DB_USER,
 )
+from opentelemetry.semconv._incubating.attributes.net_attributes import (
+    NET_PEER_NAME,
+    NET_PEER_PORT,
+    NET_TRANSPORT,
+)
+from opentelemetry.semconv.attributes.db_attributes import DB_SYSTEM_NAME
+from opentelemetry.semconv.trace import DbSystemValues, NetTransportValues
 from opentelemetry.trace import SpanKind
 from opentelemetry.trace.status import Status, StatusCode
 
 
 def _hydrate_span_from_args(connection, query, parameters) -> dict:
     """Get network and database attributes from connection."""
-    span_attributes = {
-        SpanAttributes.DB_SYSTEM: DbSystemValues.POSTGRESQL.value
-    }
+    span_attributes = {DB_SYSTEM_NAME: DbSystemValues.POSTGRESQL.value}
```

Comment:
```suggestion
    span_attributes = {DB_SYSTEM: DbSystemValues.POSTGRESQL.value}
```

This should be DB_SYSTEM as it was